3drepo/3drepo.io
# ISSUE #3334 - Processing states in Federations

This fixes #3334

#### Description
- Adds **status** and **description** to the federation dashboard list
- If the federation has a status of OK the description is shown, otherwise the status is displayed

#### Test cases
- View federations which have different statuses
- Create new federations which have different statuses and see them added to federations list with correct status/description
